demo-ui/app/uk/gov/ons/addressIndex/demoui/views/singleMatch.scala.html

```diff
@@ -9,6 +9,27 @@
 
 @main(title = messages("single.title")) {
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+
+    <script>
+        $(function () {
```

Comment:
`$(function(){})` is an alias for `$(document).ready(function(evt) {})`
http://stackoverflow.com/questions/7642442/what-does-function-do